src/storage/txn/scheduler.rs

```diff
@@ -479,6 +479,12 @@ fn process_write_impl(cid: u64,
             }
         }
         Command::Commit { ref keys, lock_ts, commit_ts, .. } => {
+            if lock_ts > commit_ts {
```

Comment:
It's invalid when `lock_ts == commit_ts` now. I make it valid here as it may be valid some day if we support transaction with 1 pc. @BusyJay